os_command_injection/DipakAhire0523_SAST-Project/command-injection.js

```
@@ -0,0 +1,78 @@
+  const fullPath = `./public/${file}`;
+  
+  res.download(fullPath);
+});
+
+// VULNERABLE: Code injection via eval
+app.post('/calculate', (req, res) => {
+  const expression = req.body.expression;
+  
+  try {
+    // BAD: Using eval with user input
+    const result = eval(expression);
```

Biome (2.1.2)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)